core/primitives/sector_file/sector_file.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_SECTOR_FILE_HPP
+#define CPP_FILECOIN_SECTOR_FILE_HPP
+
+#include <primitives/sector/sector.hpp>
+#include <string>
+
+using fc::primitives::sector::SectorId;
+
+namespace fc::primitives::sector_file {
+
+  class SectorFileType {
+   public:
+    SectorFileType(int type);
+
+    operator int() const;
+
+    SectorFileType &operator=(int rhs);
+
+    std::string string() const;
+
+    bool has(const SectorFileType &single_type) const;
+
+   private:
+    int data_;
+  };
+
+  enum SectorFileTypes {
+    FTUnsealed = 1,
+    FTSealed,
+    FTCache,
```

Comment:
This enum is for bitmask
```suggestion
    FTUnsealed = 1,
    FTSealed = 2,
    FTCache = 4,
```

Reply:
Fixed